Makefile.common

```diff
@@ -0,0 +1,54 @@
+INCFLAGS := -I$(CORE_DIR) -I$(CORE_DIR)/arm
+
+SOURCES_C :=   $(CORE_DIR)/arm/arm.c \
+					$(CORE_DIR)/arm/isa-thumb.c \
+					$(CORE_DIR)/arm/isa-arm.c \
+					$(CORE_DIR)/arm/decoder-arm.c \
+					$(CORE_DIR)/arm/decoder-thumb.c \
+					$(CORE_DIR)/arm/decoder.c \
+					$(CORE_DIR)/debugger/debugger.c \
+					$(CORE_DIR)/debugger/memory-debugger.c \
```

Comment:
I'm not sure if the debugger folder is necessary. If it is, I should fix it to not be.